python/cuopt/cuopt/linear_programming/problem.py

```
@@ -16,6 +16,100 @@
 import warnings
 
 
+# ---- Display helpers for __str__/__repr__ ----
+
+_SENSE_SYMBOLS = {LE: "<=", GE: ">=", EQ: "=="}
```

Ruff (0.15.15)

[error] Undefined name `LE`

(F821)